bezier-rs/lib/src/lib.rs

```diff
@@ -331,6 +331,41 @@ impl Bezier {
 		};
 		bezier_starting_at_t1.split(adjusted_t2)[t2_split_side]
 	}
+
+	/// Returns two lists of `t`-values representing the local extrema of the `x` and `y` parametric curves respectively
+	/// The local extrema are defined to be points at which the derivative of the curve is equal to zero
+	fn _local_extrema(&self) -> [Vec<f64>; 2] {
```

Comment:
Underscore implies unused. Can you pick a more descriptive name for this?

Reply:
Or better yet, define this helper function as a closure within its using function. (Unless you use it in multiple functions throughout the library.)

Reply:
I'm just going to rename it, as I believe we may need to use it elsewhere.